airflow/contrib/hooks/gcp_bigtable_hook.py

```diff
@@ -25,9 +25,12 @@
 from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
 
 
-class BigtableHook(GoogleCloudBaseHook):
+class (GoogleCloudBaseHook):
     """
```

Comment:
Something terrible here to stay.

Reply:
Yeah. Terrible things happen sometimes :). Fixed.